app/scripts/filters/resources.js

```diff
@@ -120,16 +120,29 @@ angular.module('openshiftConsole')
       });
       return nameCount;
     }
-    return function (resource, projects){
+    function escapeLessThanGreaterThan(str) {
+      return str.
+        replace(/</g, '&lt;').
+        replace(/>/g, '&gt;');
+    }
```

Comment:
You can use `_.escape`, although I suggest adding it as a separate filter to util.js we can call anywhere.

```js
  .filter('escape', function() {
    return _.escape;
  });
```

then

```html
<div ng-bind-html="project | uniqueDisplayName : projects | escape | highlight : $select.search"></div>
```